kube/kd/deployment.yaml

```diff
@@ -136,7 +136,7 @@ spec:
               readOnly: true
           ports:
             - name: https
-              containerPort: 10443
+              containerPort: {{.PORT}}
```

Comment:
This isn't required- as we communicate with it through a service (which we shouldn't need). The device uses the default https port of 10443, which is why this was set as such.

Reply:
Reverted.